src/app/login/login.component.ts

```diff
@@ -13,14 +14,21 @@ export class LoginComponent {
     password: new FormControl('', Validators.required),
   });
 
-  constructor(private loginService: AuthenticationService) {}
+  constructor(
+    private loginService: AuthenticationService,
```

Comment:
Service injection naming should follow actual service name
```suggestion
    private authService: AuthenticationService,
```